Sources/StreamChatUI/ChatMessageList/ChatChannelHeaderView.swift

```diff
@@ -0,0 +1,144 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The view that displays channel information on the message list header
+public typealias ChatChannelHeaderView = _ChatChannelHeaderView<NoExtraData>
+
+/// The view that displays channel information on the message list header
+open class _ChatChannelHeaderView<ExtraData: ExtraDataTypes>:
+    _View,
+    ThemeProvider,
+    _ChatChannelControllerDelegate {
+    /// Controller for observing data changes within the channel.
+    open var channelController: _ChatChannelController<ExtraData>?
+
+    /// The user id of the current logged in user.
+    open var currentUserId: UserId? {
+        channelController?.client.currentUserId
+    }
+
+    /// Timer used to update the online status of member in the channel.
+    open var timer: Timer?
```

Comment:
```suggestion
    open var timer: Timer? {
     didSet { oldValue?.invalidate() }
    }
```

Reply:
This timer is only created one time, so I don't think this is needed 🤔 Unless the customer change this 🤔

Reply:
✅

Reply:
but a single timer being active in the moment is what we want, right? 😄 With `makeTime` moved to `setUp` it became less fragile to errors on the client side but a different time can still be assigned. At the same time, `private(set)` also works fine